README.rst

```diff
@@ -242,7 +242,7 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
-``OPENWISP_MONITORING_AUTO_GRAPHS``
+``OPENWISP_MONITORING_AUTO_CHARTS``
```

Comment:
Changing this setting name and updating to the latest version will add some friction to the upgrade. I would keep this name and if we want to keep consistency I would add a deprecation warning.

@nemesisdesign WDYT?

Reply:
@PabloCastellano, the GSoC project's name was `Improving OpenWISP Monitoring towards its first release`. Hence, I don't think it's been officially released yet but maybe it might be out for `beta` testing in which case I might be wrong :sweat_smile:

Reply:
That's right. Let's wait for Federico who has more details on who is actually using this in production

Reply:
Thanks Pablo for the concern, in this case this change does not affect negatively already deployed systems so we can proceed.